### ISSUE TITLE
WHAT THE?

### ISSUE THREAD

**Issue by SonicSuperFan** [AlexField442/Sonic-2-Simon-Wai-Disassembly#7]
my sonic is lining
![image](https://github.com/user-attachments/assets/4b7a1660-8884-4373-9534-a36d803d17ec)

**Comment by FourHawk** [comment]
Can't seem to recreate it on my end, you editing the rom by any chance?

**Comment by SonicSuperFan** [comment]
no. all i did was make wood zone a bit mroe complete (added more chunks and
removed old chunks that are broken. even happens on all zones!)

**Comment by FourHawk** [comment]
Seems like a DMA issue

**Comment by SonicSuperFan** [comment]
> Seems like a DMA issue

whats a dma issue?

**Comment by FourHawk** [comment]
DMA "Direct Memory Access" is how the mega drive uploads Sonic's graphics to VRAM, there's a really annoying issue that can show up where if a DMA transferrer crosses a 128KiB ROM boundary, things can get a little corrupted, this is an actual hardware issue, so there isn't a true fix you just have to get around it, best way i can think of is to move sonic's art to another spot in the rom

**Comment by SonicSuperFan** [comment]
But how?

**Comment by FourHawk** [comment]
Find `ArtUnc_Sonic:	BINCLUDE	"art/uncompressed/Sonic's art.bin"` and move it to somewhere else in the rom

**Comment by SonicSuperFan** [comment]
How to move it?
